src/index.html

```diff
@@ -3,14 +3,24 @@
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!--Search_Bar_Big-->
+    <div class="search__container">
+      <div class="search_block" data-qa="big">
+        <div data-qa="keypress">
+          <input class="search_item search_block_big search--focused" placeholder="Try &quot;Los Angeles&quot;">
+        </div>
+      </div>
```

Comment:
Why did you use only tag div here?
You need to use the semantic tag form, label and input, so need to remove divs